hyperspy/misc/eds/utils.py

```diff
@@ -243,6 +247,72 @@ def take_off_angle(tilt_stage,
                                   + math.sin(a) * math.sin(c)))
 
 
+def xray_lines_model(elements=['Al', 'Zn'],
```

Comment:
Why does `elements` default to `['Al', 'Zn']` ? `weight_percents` should possibly default to `None`, but can be autofilled to an even distribution depending on the number of elements passed.